app-tasks/rf/src/rf/models/base.py

```diff
@@ -44,5 +46,10 @@ def create(self):
         url = '{HOST}{URL_PATH}'.format(HOST=self.HOST, URL_PATH=self.URL_PATH)
         session = get_session()
         response = session.post(url, json=self.to_dict())
-        response.raise_for_status()
-        return self.from_dict(response.json())
+        if response.status_code != 409:
```

Comment:
The `409` behavior only applies to creating duplicate Scenes, right? I'm not sure that we want to ignore it in all circumstances, so this might be more appropriate as an override on the `Scene` subclass rather than putting it on the `BaseModel`.

Also, I think it would be more Pythonic to do something along the lines of:
```python
try:
    response.raise_for_status()
except HTTPError:
    # check if it was a 409, reraise if not.